src/tower/resources/environment.yaml

```diff
@@ -15,5 +15,8 @@ TOWER_LICENSE: "!Ref 'TowerLicense'"
 FLYWAY_LOCATIONS: "classpath:db-schema/mysql"
 TOWER_GOOGLE_CLIENT: "!Ref TowerGoogleClientId"
 TOWER_GOOGLE_SECRET: "!Ref TowerGoogleSecret"
+TOWER_OIDC_CLIENT: "!If [!Ref HasTowerOidcClient, !Ref TowerOidcClientId, !Ref AWS::NoValue]"
+TOWER_OIDC_SECRET: "!If [!Ref HasTowerOidcClient, !Ref TowerOidcSecret, !Ref AWS::NoValue]"
+TOWER_OIDC_ISSUER: "!If [!Ref HasTowerOidcClient, !Ref TowerOidcIssuer, !Ref AWS::NoValue]"
```

Comment:
@zaro0508: I'm trying to set these environment variables only when the `HasTowerOidcClient` condition evaluates to true. I don't think setting the `Value` of an environment variable to `AWS::NoValue` will have the intended effect.

Reply:
Never mind. I found a simpler way. Let me push that commit.